### PR TITLE
Update Loading effect

### PR DESCRIPTION
**Pull request by rondoor124**
- update logic for the loading effect
- adding event handler
- restructure
- fix some unexpected issue.
[issue remained]
- the html video player is crashed on the low performance 
- doesn't work smoothly on the low internet traffic environment

### CODE REVIEW

File: web/player.ts [hunk 1]
```diff
@@ -209,37 +181,52 @@ export default class Player {
...
-        left: '0',
-        width: '1px'
-      },
-      playbackRate: 1
+      playbackRate: 1,
+      muted: "muted",
     });
     this._video.setAttribute('crossorigin', '');
     this._video.setAttribute('preload', 'auto');
+    this._video.setAttribute('playsInline', 'playsInline');
```
Comment (speigg) [inline web/player.ts:201]: playsinline attribute should be all lowercase, and the value should be an empty string

File: web/player.ts [hunk 2]
```diff
@@ -209,37 +181,52 @@ export default class Player {
...
+    this._video.setAttribute('muted', 'muted');
```
Comment (speigg) [inline web/player.ts:202]: value should be empty string

**Review by speigg — CHANGES_REQUESTED** [review]

**Comment by speigg** [comment]
When I test this branch w/ the [Volumetric Player](https://github.com/XRFoundation/UniversalVolumetric-Demo) the videoStatus stays on VideoStatusEnum.Set. 

My steps:

In Universal-Volumetric repo (this branch):
```
npm run build
npm link
```

In Volumetric Player repo:
```
npm i
npm link volumetric
npm run dev
```

Then I try the sample volumetric videos, and they don't play. Does the player need to be updated?

**Comment by rondoor124** [comment]
Thanks for the review!
I did not try to check with Volumetric Player, I will check soon.

**Pull request merged**